joomla/jissues
# Support new GitHub webhook events (Ref #776)

Pull Request for Issue #776

#### Summary of Changes

Supports new GitHub webhooks for editing of issue, PR, and comment contents and deleting comments

For issues and pull requests, when the `edited` webhook action is triggered a slimmed down version of our update process will run which specifically only grabs the edited fields and logs an edited activity to the table.  Truthfully not much different than our update handler is in general, but if we decide to make other changes later on (part of the reason we were always updating the title/body on all events is because of the lack of this webhook action) it's not too difficult.

For comments, when the `edited` webhook action is triggered the comment's contents should be updated (no other changes happen in the database; in the UI we may consider adding an edited comment similar to how GitHub displays them).  When the `deleted` webhook action is triggered the corresponding activity record is deleted.

#### Testing Instructions

Have a repository push issue, pull request, and issue/PR comment webhooks to an install of the tracker and ensure the events are processed.
